src/qt/bitcoin.cpp

```diff
@@ -521,7 +521,7 @@ int GuiMain(int argc, char* argv[])
     try {
         node->selectParams(gArgs.GetChainName());
     } catch(std::exception &e) {
-        node->initError(strprintf("%s\n", e.what()));
+        node->initError(strprintf(_("%s\n"), e.what()));
```

Comment:
Here and the two other instances: The The gui is not yet subscribed to core signals and can't pick up the translation, which is why it is manually translated below.

Reply:
Correct. Fixed.